### PR TITLE
Migrate Avatar Image; closes #2359.

### LINKED ISSUE #2359: Migrate Avatar Image
AvatarImage displays WordPress User gravatar images

- see `assets\ZZZ\components\ui\image\avatar.js`

- move to `assets\src\application\ui\display\avatar`

- convert to TS

- can use https://ant.design/components/avatar/

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- Closes https://github.com/eventespresso/event-espresso-core/issues/2359

### CODE REVIEW

File: assets/src/application/ui/layout/espressoTable/EspressoTable.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { pathOr, propOr } from 'ramda';
```
Comment (nicolad) [inline assets/src/application/ui/layout/espressoTable/EspressoTable.tsx]: `propOr` wasn't used

File: assets/src/application/ui/display/Avatar.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { Avatar as DefaultAvatar, Badge } from 'antd';
```
Comment (tn3rb) [inline assets/src/application/ui/display/Avatar.tsx]: looks like you forgot to use this alias

File: assets/src/application/ui/display/Avatar.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+	if (userName) {
+		const size = avatarProps.size || 'large';
+
+		return (
+			<Avatar {...avatarProps} size={size}>
+				{userName}
+			</Avatar>
+		);
+	}
```
Comment (nicolad) [inline assets/src/application/ui/display/Avatar.tsx]: these 2 are based on antd demos

File: assets/src/application/ui/display/Avatar.tsx (hunk 2 above)
Comment (nicolad) [reply assets/src/application/ui/display/Avatar.tsx]: my brain is tired now :)

**Pull request merged**